src/NodeCompiler/Exception/UnableToCompileNode.php

```diff
@@ -66,11 +69,25 @@ public static function becauseOfNotFoundConstantReference(
         return $exception;
     }
 
+    private static function getFileName(CompilerContext $fetchContext): string
+    {
+        return $fetchContext->getFileName() ?? '""';
```

Comment:
Is it worth stating "no file" rather than "" in this case? Not important, since we didn't even have the file name before